scripts/setupvars/setupvars.ps1

```diff
@@ -0,0 +1,118 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Arguments parsing
+param (
+    [string]$python_version
+)
+
+$Env:INTEL_OPENVINO_DIR = Split-Path $MyInvocation.MyCommand.Path
+
+$Env:InferenceEngine_DIR = "$Env:INTEL_OPENVINO_DIR/runtime/cmake"
+$Env:ngraph_DIR = "$Env:INTEL_OPENVINO_DIR/runtime/cmake"
```

Comment:
both `InferenceEngine_DIR` and `ngraph_DIR` are obsolete and will be removed in 2024.0 (current master)

Suggest dropping these variables.